packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -311,6 +311,10 @@ function MongoNav({
     `${hosts.cloud}/v2/${projectId}#`;
   const constructProjectURL = constructProjectURLProp ?? defaultProjectURL;
 
+  const filteredProjects = data?.projects.filter(project => {
+    return project.orgId === data?.currentProject?.orgId;
```

Comment:
```suggestion
    return project.orgId === data.currentProject?.orgId;
```
Should be fine as we already validated that data exists in the first line.

Reply:
you're right 🤦‍♀